pkg/gui/context/sub_commits_context.go

```diff
@@ -71,12 +73,45 @@ func NewSubCommitsContext(
 			selectedCommitSha,
 			startIdx,
 			endIdx,
-			shouldShowGraph(c),
+			// Don't show the graph in the left/right view; we'd like to, but
+			// it's too complicated:
+			shouldShowGraph(c) && viewModel.GetRefToShowDivergenceFrom() == "",
```

Comment:
This is really too bad; I'd like to have the graph in the left/right view, too. But I tried, and it's too hard to do; not only would we have to _render_ the graph in two sections, but we'd actually have to calculate two separate sets of pipe sets; and then there's this global cache for these. I'll leave this as a future improvement.